### DISCUSSION TITLE
Use create-turbo with Javascript instead of TypeScript

### DISCUSSION THREAD

**Discussion by runoncedev** [vercel/turborepo#787] [Help] [answered]
Is it possible to start a new project using `npx create-turbo@latest` with Javascript instead of TypeScript for the example apps (`docs` and `web`)?

Looked into the docs and tried `npx create-turbo@latest --help` without any luck.

**Comment by samchouse** [comment, marked as answer]
Not right now, we only generate TS projects. You can remove any TS configs or setup a JS project manually if you want to use JS.

**Reply by runoncedev** [reply]
Yes, that's what I did. I'm having some trouble trying to include a `create-react-app` though, is giving me this error when I try to run `npm run dev`:

> Loading PostCSS "postcss-preset-env" plugin failed: Cannot find module 'postcss-preset-env'

I just asked on StackOverflow with additional details and a reproducible repo: https://stackoverflow.com/questions/71315857/cannot-find-module-postcss-preset-env-error-using-create-react-app-inside-a-np
